azure/deploy-to-azure/azure-integration/CreateUIDefinition-managementgroups.json

```diff
@@ -134,7 +134,7 @@
                 "type": "Microsoft.Common.TextBlock",
                 "visible": true,
                 "options": {
-                  "text": "To fill out the fields below, copy the API and App Keys from your Datadog org. If you launched this template from the Azure tile, you can copy the keys you selected directly from there. Otherwise you can find your API and App Keys in the Access section of the Organization Settings.",
+                  "text": "To fill out the fields below, copy the API and App Keys from your Datadog org. If you launched this template from the Azure tile, refer back to the tile to select and copy the keys. Otherwise you can find your API and App Keys in the Access section of the Organization Settings.",
```

Comment:
Do we use the term "tile" to describe the page to customers? I wonder if we should be saying "Azure integration config page" or something, especially with the coming revamp